app/views/layouts/madeline/template.html.erb

```diff
@@ -0,0 +1,18 @@
+<%# Defines content template variables based on page displayed %>
+<%= render "layouts/madeline/page_variables" %>
```

Comment:
I'm confused about the purpose of the page_variables file. This looks like the kind of logic that may belong in a controller rather than a view.